hal/ia32/cpu.c

```diff
@@ -283,9 +292,25 @@ void hal_longjmp(cpu_context_t *ctx)
 		popw %%fs;\
 		popw %%es;\
 		popw %%ds;\
+		testl %[cr0ts], 108(%%esp); /* FPU_CONTEXT_SIZE */\
+		movl %%eax, 108(%%esp);	   /* FPU_CONTEXT_SIZE */\
```

Comment:
Instead of magic number.
```suggestion
		testl %[cr0ts], %c3(%%esp); /* FPU_CONTEXT_SIZE */\
		movl %%eax, %c3(%%esp);	   /* FPU_CONTEXT_SIZE */\
```